support/ci/promote_rc.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -e
+
+if [ -z ${HAB_AUTH_TOKEN+x} ]; then
+  echo "HAB_AUTH_TOKEN var is unset"
+  exit 1
+fi
+if [ -z ${BINTRAY_USER+x} ]; then
+  echo "BINTRAY_USER var is unset"
+  exit 1
+fi
+if [ -z ${BINTRAY_KEY+x} ]; then
+  echo "BINTRAY_KEY var is unset"
+  exit 1
+fi
+
+HAB_VERSION=$(cat VERSION)
+PROMOTE_CHANNEL=stable
+RC_CHANNEL=rc-$HAB_VERSION
+RC_PKGS_JSON=$(curl https://bldr.habitat.sh/v1/depot/channels/core/$RC_CHANNEL/pkgs)
+RC_PKGS_JSON_DATA=$(echo $RC_PKGS_JSON | jq -c -r '.data | .[]')
+TOTAL_COUNT=$(echo $RC_PKGS_JSON | jq -r '.total_count')
+END=$(echo $RC_PKGS_JSON | jq -r '.range_end')
+
+if [ $(($END+1)) !=  $TOTAL_COUNT ]; then
+  echo "There are multipe pages of releases. Consider deleting the channel and then rebuild."
```

Comment:
Maybe add a blurb about our default page size being 50 right now, and we are in no way expecting there to be anywhere _near_ that many packages in a normal release channel.

Just so Future Us knows why we didn't bother implementing pagination here.